frontend/src/components/footer.js

```diff
@@ -72,7 +75,6 @@ const Footer = () => {
                   {connectList.map((item, index) => (
                     <a
```

Comment:
Can we use Link gatsby component for internal links and just use normal anchor (a) for external links

Reply:
yes im going to merge this pr first and then in my links refactor pr ill update this